### ISSUE TITLE
The windrose program in CLIMSOFT Version 4?

### ISSUE THREAD

**Issue by rdstern** [opencdms/Climsoft#328]
This is a separate programme which is free, but needs a licence key.  Do we need it?

The installation is complicated enough without this.  It seems odd to have this feature just for a special graph for wind data.  There are no graphs in CLIMSOFT yet for other types of data.

It is also a limited wind rose.  I have yet to check carefully, but I think it only does one wind-rose at a time. 

The clifro package in R is a web portal to the New Zealand National Climate Database.  It is produced by the New Zealand Met Service I think.  It is one source of a windrose in R and we have added a simple version to R-Instat.  It has (automatically) the facility to plot facets, i.e. to be able to provide a windrose for (say) each of 12 months in the same graph, or for a set of stations.

Perhaps we could discuss this situation in Rwanda.  Changing the windrose capabilities would have a lower priority to me than other suggestions, so I await revised milestones for CLIMSOFT.  (My use of github doesn't give me access to these facilities, so someone with more privileges will have to do this.)

**Comment by smachua** [comment]
@rdstern I agree with you that we can discuss windrose and other graphics later and compare the available resources for that purpose.

**Comment by isedwards** [comment]
During the Kitale workshop we decided to remove WRPlot from the installation program.

- [ ] Check that Climsoft provides sufficient help for users when WRPlot is not found so that they can install the software if they want to use it.
